scripts/startLocalCluster.sh

```diff
@@ -230,6 +230,7 @@ start() {
       --log.level $LOG_LEVEL_CLUSTER
       --server.descriptors-minimum 0
       --javascript.allow-admin-execute true
+      --http.trusted-origin all
```

Comment:
Not sure if this should be bundled with this PR.

Reply:
Double-checked with @mpoeter . It's fine to keep it in this PR.